identity/src/main/java/com/stripe/android/identity/ui/AddressSection.kt

```diff
@@ -0,0 +1,105 @@
+package com.stripe.android.identity.ui
+
+import androidx.compose.material.Text
+import androidx.compose.material.TextButton
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.LaunchedEffect
+import androidx.compose.runtime.collectAsState
+import androidx.compose.runtime.derivedStateOf
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.testTag
+import androidx.navigation.NavController
+import com.stripe.android.core.model.Country
+import com.stripe.android.identity.R
+import com.stripe.android.identity.navigation.CountryNotListedDestination
+import com.stripe.android.identity.navigation.navigateTo
+import com.stripe.android.identity.networking.Resource
+import com.stripe.android.identity.networking.models.RequiredInternationalAddress
+import com.stripe.android.identity.viewmodel.IdentityViewModel
+import com.stripe.android.uicore.elements.AddressElement
+import com.stripe.android.uicore.elements.IdentifierSpec
+import com.stripe.android.uicore.elements.SectionElement
+import com.stripe.android.uicore.elements.SectionElementUI
+
+/**
+ * Section to collect User's Address.
+ */
+@Composable
+internal fun AddressSection(
+    identityViewModel: IdentityViewModel,
```

Comment:
We should avoid passing this down that much. Maybe you can retrieve it using `androidx.lifecycle.viewmodel.compose.viewModel()`?

Reply:
as discussed above, tracked in https://jira.corp.stripe.com/browse/IDPROD-5412